Function closures in addConversionFuncs is bound to same 'kind' variable

sudhi-vm opened the pull request:
[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/.github/PULL_REQUEST_TEMPLATE.md?pixel)]()

Fixes #28614

This PR has a simple fix to move creating the field label conversion func
for ObjectMeta fields into another local method.

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.kubernetes.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/28621)

<!-- Reviewable:end -->

smarterclayton commented:
Can you get your CLA sorted out?

ok to test

sudhi-vm commented:
I signed it!

smarterclayton commented:
@thockin does this interfere with your generated files changes?  If so I
will hold.

On Fri, Jul 8, 2016 at 2:36 AM, googlebot <notifications@github.com> wrote:

> CLAs look good, thanks!
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28621#issuecomment-231286317>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p1n5u87RotCfu7CzMe-Iw58w90lqks5qTe_9gaJpZM4JHYg6>
> .
>

thockin commented:
Thanks for checking in.  The go2idl part of my change is in now.  I still need to get the Makefile parts in, but that's far less broadly impacted by rebases :)

smarterclayton commented:
@k8s-bot test this issue

smarterclayton commented:
Can you please rebase?

sudhi-vm committed:
Function closures in addConversionFuncs is bound to same 'kind' variable

Fixes #28614

This PR has a simple fix to move creating the field label conversion func
for ObjectMeta fields into another local method.

sudhi-vm commented:
@smarterclayton I noticed that this has been already fixed in 2 of 3 files in my original PR. I rebased & updated the PR to cover the last file.
